src/test/run-make-fulldeps/coverage-reports/expected_show_coverage.loops_branches.txt

```diff
@@ -12,27 +12,51 @@
    12|       |                while true {
    13|       |                }
    14|      1|            }
-   15|      1|            write!(f, "error")?;
-                                            ^0
+   15|      0|            write!(f, "error")?;
```

Comment:
any ideas on why the whole line is marked as uncovered now?

Reply:
Great catch. Yeah, I just fixed it and will update the PR commit after validating the test suite again.

I added a check for expressions that subtract from zero, and I thought I could drop expressions that do that, but that drops their code region too, of course (and shouldn't). That seems to be the cause.

Reply:
done